adr_arch_pattern.md

```diff
@@ -0,0 +1,15 @@
+## ADR - Architectural pattern for components.
+
+### Status
+Proposed
+
+### Context
+In order to build the components identified from event storming, an architectural style should be chosen.
+
+### Decision
+For identifying the architectural pattern, the ility requirements of individual services were identified. 
+
+//TODO: Add screenshot of ilities and arch pattern
+
+### Consequences
+It is decided that we can use Microservices pattern for the system.
```

Comment:
@avinash-zemoso , this should go under decision. Add drawbacks of using microservices in consequences. ANd how to mitigate them